misk/src/main/kotlin/misk/metrics/Metrics.kt

```diff
@@ -57,4 +56,4 @@ class Metrics @Inject internal constructor(val registry: CollectorRegistry) {
   }
 }
 
-internal val Histogram.Child.Value.count get() = buckets.max()?.toInt() ?: 0
+internal val io.prometheus.client.Histogram.Child.Value.count get() = buckets.max()?.toInt() ?: 0
```

Comment:
I don't think this is required anymore, since we added `count` to the `Histogram` interface

Reply:
either way, nothing in this class should have a reference to a prometheus Histogram